crates/fj-viewer/src/graphics/renderer.rs

```diff
@@ -183,6 +186,13 @@ impl Renderer {
         let pipelines =
             Pipelines::new(&device, &bind_group_layout, color_format);
 
+        let navigation_cube_renderer = NavigationCubeRenderer::new(
+            &device,
+            &queue,
+            &surface_config,
+            800.0 / 600.0,
```

Comment:
I don't understand why we need to pass an aspect ratio here, and why it is set to this value specifically.

Everything seems to work fine, so it's not too much of a problem, but I figure this could be made clearer. Since the aspect ratio is set to a constant anyway, maybe we should move this value to where it's actually used and add a comment there, explaining why that specific value was chosen.